modules/manually-create-identity-access-management.adoc

```diff
@@ -0,0 +1,178 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/manually-creating-iam.adoc
+// * installing/installing_azure/manually-creating-iam-azure.adoc
+// * installing/installing_gcp/manually-creating-iam-gcp.adoc
+
+ifeval::["{context}" == "manually-creating-iam-aws"]
+:aws:
+endif::[]
+ifeval::["{context}" == "manually-creating-iam-azure"]
+:azure:
+endif::[]
+ifeval::["{context}" == "manually-creating-iam-gcp"]
+:google-cloud-platform:
+endif::[]
+
+[id="manually-create-iam_{context}"]
+== Manually create IAM
+
+The Cloud Credential Operator can be put into manual mode prior to installation
+in environments where the cloud identity and access management (IAM) APIs are
+not reachable, or the administrator prefers not to store an admin level
+credential secret in the cluster `kube-system` namespace.
+
+.Procedure
+
+. Run the {product-title} installer to generate manifests:
++
+----
+$ openshift-install create manifests --dir=mycluster
+----
+
+. Insert a ConfigMap into the manifests directory so that the Cloud Credential
+Operator will be be placed in manual mode:
++
+----
+$ cat <<EOF > mycluster/manifests/cco-configmap.yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cloud-credential-operator-config
+  namespace: openshift-cloud-credential-operator
+  annotations:
+    release.openshift.io/create-only: "true"
+data:
+  disabled: "true"
+EOF
+----
+
+. Remove the admin credential secret created using your local cloud credentials.
```

Comment:
Throughout, suggest reassessing whether "admin" should be spelled out "administrator" or marked up `admin`.